contrib/maildisguise.settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "_config_name": "maildisguise.settings",
```

Comment:
Shouldn't the directory name be "config" instead of contrib? ;) Never mind, I got the point.

Reply:
Ah yes! Sorry, typo :smile: